theme/default/css/css.css

```diff
@@ -3790,27 +3794,26 @@ select[multiple].input-group-sm > .input-group-btn > .btn {
     display: none;
   }
 }
+
 .navbar-nav {
-  margin: 7.5px -15px;
+  margin: 0;
```

Comment:
This seems to corrupt the normal nav-bar. Take a look at the base tools, you will see some ugly navigation bars (without buttons the items are all near). 
We should keep the bootstrap-theme more clear as possible, so it can be used everywhere and keep the look and feel.